src/RingDecomposerLib/CMakeLists.txt

```diff
@@ -28,4 +28,6 @@ build_library("${SHARED_LIBRARY}" "SHARED")
 build_library("${STATIC_LIBRARY}" "STATIC")
 
 install(FILES RingDecomposerLib.h
-        DESTINATION include)
+RDLapsp.h    RDLcycleFams.h   RDLdimacs.h   RDLgraph.h    RDLinfo.h      RDLstack.h   RDLtesting.h
```

Comment:
I'm not happy with this one because I left these one out from the interface on purpose. And you don't really need them. I've fixed the integration of the RingDecomposerLib into the RDKit (the custom deleter was missing for the boost::shared_ptr so incomplete types will be OK and also the memory is freed correctly) so you won't need these files accessible. I've made a PR to your *private* rdkit repo (branch feat/URF_integration) because you closed your PR on the official repo.

Reply:
makes sense. I will remove them

Reply:
done